### PR TITLE
feat(task-settings-template): Added Task Settings Template endpoints

### LINKED ISSUE #106: Tasks Settings Templates API support
**New API endpoints:**

- [api.projects.tasks.settings-templates.getMany](https://developer.crowdin.com/api/v2/#operation/api.projects.tasks.settings-templates.getMany)
- [api.projects.tasks.settings-templates.post](https://developer.crowdin.com/api/v2/#operation/api.projects.tasks.settings-templates.post)
- [api.projects.tasks.settings-templates.get](https://developer.crowdin.com/api/v2/#operation/api.projects.tasks.settings-templates.get)
- [api.projects.tasks.settings-templates.delete](https://developer.crowdin.com/api/v2/#operation/api.projects.tasks.settings-templates.delete)
- [api.projects.tasks.settings-templates.patch](https://developer.crowdin.com/api/v2/#operation/api.projects.tasks.settings-templates.patch)

**Crowdin Enterprise** endpoints are almost the same, but there is some difference in template config:

- [api.projects.tasks.settings-templates.getMany](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.getMany)
- [api.projects.tasks.settings-templates.post](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.post)
- [api.projects.tasks.settings-templates.get](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.get)
- [api.projects.tasks.settings-templates.delete](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.delete)
- [api.projects.tasks.settings-templates.patch](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.patch)

### PR DESCRIPTION
**Pull request by tchapuis**
Closes #106

### CODE REVIEW

File: src/CrowdinApiClient/Api/TaskApi.php [hunk 1]
```diff
@@ -152,4 +153,85 @@ public function userTaskArchivedStatus(int $taskId, int $projectId, bool $isArch
...
+     * @link https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.settings-templates.getMany API Documentation Enterprise
+     *
+     * @param int $projectId
+     * @param array $params
+     * integer $params[limit]  [ 1 .. 500 ] Default: 25<br>
+     * integer $params[offset]  >= 0 Default: 0<br>
+     *
+     * @return ModelCollection
+     */
+    public function listSettingsTemplate(int $projectId, array $params = []): ModelCollection
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/TaskApi.php:169]:
```suggestion
    public function listSettingsTemplates(int $projectId, array $params = []): ModelCollection
```

**Comment by andrii-bodnar** [comment]
@tchapuis looks good to me, thank you! Could you please address the discussion above so I can merge the PR?

**Comment by andrii-bodnar** [comment]
@tchapuis please update tests as well 🙂

**Comment by tchapuis** [comment]
> @tchapuis please update tests as well 🙂

Yes sorry, I was a bit to fast 😅

**Review by andrii-bodnar — APPROVED** [review]
Thank you!

**Pull request merged**